e2e/tests/room/01-createroom.spec.ts

```diff
@@ -1,9 +1,21 @@
 import { device, waitFor, element, by, expect } from 'detox';
 
-import { tapBack, navigateToLogin, login, platformTypes, TTextMatcher, tapAndWaitFor } from '../../helpers/app';
+import { tapBack, navigateToLogin, login, platformTypes, TTextMatcher, tapAndWaitFor, sleep } from '../../helpers/app';
 import { createRandomUser } from '../../helpers/data_setup';
 import random from '../../helpers/random';
 
+async function searchRoomNonLatin(room: string, roomTestID: string) {
```

Comment:
Why do you need this function?
If `searchRoom` isn't enough, you should not create another one in a separate file.
You should consider changing its behavior.

Reply:
Why? This is specific for this type of test, I'm testing if the searching is case insensitive for non-ASCII